### PR TITLE
chore(RHTAPWATCH-346 RHTAPWATCH-347): add sop links to 2 pod alerts

### PR DESCRIPTION
**Pull request by yftacherzog**
Adding link to SOPs to alerts CrashLoopBackOff and UnschedulablePods to provide instructions for dealing with those alerts.

### CODE REVIEW

File: rhobs/alerting/prometheus.pod_alerts.yaml [hunk 1]
```diff
@@ -15,16 +15,24 @@ spec:
...
       annotations:
-        summary: "Pod {{ $labels.pod }} is crash looping"
-        description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') on cluster {{ $labels.source_cluster }} for more than 15 minutes."
+        summary: Pod {{ $labels.pod }} is crash looping
+        description: >-
+          Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in
+          waiting state (reason: 'CrashLoopBackOff') on cluster
+          {{ $labels.source_cluster }} for more than 15 minutes. Related SOP:
+          https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-crashLoopBackOff.md
     - alert: PodsNotReady
```
Comment (avi-biton) [inline rhobs/alerting/prometheus.pod_alerts.yaml:36]: Don't we want to add the link to https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-PodsNotReady.md for this alert?

Comment (yftacherzog) [reply rhobs/alerting/prometheus.pod_alerts.yaml:36]: PodsNotReady is the one below.
I actually got confused with that a couple of times when working on that, so have another look to see that I'm seeing it correctly.

Comment (yftacherzog) [reply rhobs/alerting/prometheus.pod_alerts.yaml:36]: In any case, PodsNotReady is covered by RHTAPWATCH-348. We need to make sure that we're providing the correct URLs and provide them under the correct alert definition.

Comment (avi-biton) [reply rhobs/alerting/prometheus.pod_alerts.yaml:36]: I understand that `PodsNotReady` would be in other PR

**Review by avi-biton — APPROVED** [review]
lgtm

**Pull request merged**